no-computer.md

```diff
@@ -5,51 +5,79 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+A DSL for describing people by appearence.  This would be useful so that when you see someone
+you could describe more accurately what they look like to your friends. This
+could also be a useful way to report suspicious people the the authorities if
+there was a specific language for describing people in an organized manner.
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I would be helping people who have difficulty describing people.  It can be very
+difficult to explain to people what someone looks like and what differentiates
+their looks from other people.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A DSL is appropriate because it is a very focused domain: people.  A langauge is
+nice because it is a convenient way to share information.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I came up with the idea because I've heard many people describing other people
+when they can't remember someone's name and often the descriptions are very
+vague.  
 
 ### Domain
 _Describe the project's domain in five words._
 
+Describe people's appeareance precisely
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+A user would interact by the language by hearing a description or creating a
+description of someone.
 
 ### Operation (semantics)
 _What might happen when a program runs? How does a program interact with the
 user? What kinds of errors might occur, and how might they be communicated to
 the user?_
 
+I would interpret this question as someone using the DSL to describe someone.
+This would look like expressing characteristics of the individual in an ordered
+fashion.
 
 ### Expressiveness
 _What should be easy to do in this language? What should be possible, but
 difficult? What should be impossible or very difficult?_
 
+It should be easy to express different aspects of people such as eye color
+height, and other such physical attributes.  It should be impossible or very
+difficult to describe non physical attributes since this langauge should allow
+you too identify people described to you just by sight.
```

Comment:
Eye color and height is already doable in spoken language, what do you imagine would/could be different? Perhaps it can be more precise (precise height and ... hex descriptions of eye color?), but humans aren't really good at being precise without using tools to help them. I know this DSL would be that tool. I'm just curious what we could do to describe appearances more precisely without carrying around a rule and color identifier, and measuring every people we see. How could we gather and input more precise "data" into this DSL?